audio-worklet/design-pattern/lib/em-es6-module.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+
+// EXPORT_ES6 option does not work as described at
+// https://github.com/kripken/emscripten/issues/6284, so we have to
+// manually add this by '--post-js' setting when the emcc compilation.
```

Comment:
"manually add...emcc compilation" is phrased in a funny way that I don't know what it means.

Reply:
Done.